src/core/helpers.mjs

```diff
@@ -182,3 +183,20 @@ export function validatePath (input) {
   }
   return input
 }
+
+/**
+ * Download logs files from all network pods and save to local solo log directory
+ * @param k8
+ *    an instance of core/K8
+ * @returns A promise that resolves when the logs are downloaded
+ */
+export async function getNodeLogs (k8) {
+  const pods = await k8.getPodsByLabel(['fullstack.hedera.com/type=network-node'])
+  for (const pod of pods) {
+    const podName = pod.metadata.name
+    const targetDir = `${SOLO_LOGS_DIR}/${podName}`
```

Comment:
I'm thinking to include the namespace, something like: 
`${SOLO_LOGS_DIR}/${namespace}/${podName}`